homeassistant/components/alarm_control_panel/__init__.py

```diff
@@ -23,9 +23,9 @@
 
 ENTITY_ID_FORMAT = DOMAIN + '.{}'
 
-ALARM_SERVICE_SCHEMA = ENTITY_SERVICE_SCHEMA.extend({
+ALARM_SERVICE_SCHEMA = {
```

Comment:
We might even want to just inline these schemas instead of making them constants.

Reply:
In this particular case (`alarm_control_panel`), that schema is used in multiple places; are we not concerned about potentially having to update multiple places?

Reply:
oh right. good point.